orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/EvaluateDeploymentHealthTaskSpec.groovy

```diff
@@ -198,6 +205,50 @@ class EvaluateDeploymentHealthTaskSpec extends Specification {
     false              | null             || ExecutionStatus.FAILED_CONTINUE
   }
 
+  def "should return status as RUNNING when SpinnakerHttpException is thrown"() {
```

Comment:
Possible to add this test before the code change so we can see any change in behavior?

Reply:
I think there is no change in the behaviour here. This test case is to verify if the execute() method returns status as RUNNING when SpinnakerHttpException is thrown. This was the same case even with RetrofitError as well.

Reply:
Without having the test before the code change, it's hard to see that.

Reply:
With reference to this comment : https://github.com/spinnaker/orca/pull/4617#discussion_r1434434134 , moved this test case to a separate commit for a better understanding .